### ISSUE TITLE
Accessibility: Use semantic elements for emphasis

### ISSUE THREAD

**Issue by aduth** [GSA/centers-of-excellence#373]
Many pages use `<i>` (italic) and `<b>` (bold) HTML elements to represent emphasized text. In cases where emphasis is intended, the semantic elements `<em>` and `<strong>` should be used instead.

**Examples:**

- https://coe.gsa.gov/2019/04/10/cx-update-8.html
   - >Once the design research is gathered and organized, we use CX methods to identify themes and patterns—a process we call **“synthesis.”** We then generate ideas for solutions— **“ideation”**—not in unstructured brainstorming, but using a proven framework to recombine ideas into often-unexpected ways. From there, CX practitioners create solution **prototypes** that we can rapidly test, evaluate, and refine.
- https://coe.gsa.gov/coe/affordable-housing/
   - >**Illustrated the experience** of a subset of HUD’s customers, as representative of experiences common to all citizens.
   - >Facilitated HUD’s identification of **opportunities and potential solutions** for the new Office of Customer Experience’s consideration.
   - >**Demonstrated a human-centered way of working** at HUD that improves customer experience and operational efficiency.

**WCAG Success Criteria (A):** [1.3.1: Info and Relationships](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

**Relevant success criteria:** [HTML H49: Using semantic markup to mark emphasized or special text](https://www.w3.org/TR/WCAG20-TECHS/H49.html)

>This example shows how to use the `em` and `strong` elements to emphasize text. The `em` and `strong` elements were designed to indicate structural emphasis that may be rendered in a variety of ways (font style changes, speech inflection changes, etc.).

**Practical Implications:**

Many screen readers may in-fact treat `<b>` as `<strong>`, or not make stylistic or voice changes in response to the difference between the two elements, resulting in practically no effective difference.

https://accessibility.psu.edu/boldfacehtml/

>Mainstream screen readers currently treat B/I tags identically to STRONG/EM tags. That is, they are ignored unless a user specifies that they be indicated. Some users report that announcing changes can be distracting.

>In theory, screen readers could pronounce STRONG and EM in a different voice or style. However this rarely happens in practice (the same is true for B and I tags).

**Remediation Guidance:**

Prefer to use semantic HTML tags where emphasis is intended. In Markdown files, the `**` syntax for bold will often be converted by Markdown processors defaulting to the `<strong>` tag.
